coriolis_openstack_utils/conf.py

```diff
@@ -98,9 +104,10 @@
 DESTINATION_OPTS = OPENSTACK_CONNECTION_OPTS + [
     REGION_CONFIG_OPT, NETMAP_OPT, STORMAP_OPT, ENDPOINT_NAME_FORMAT_OPT,
     SKIP_OS_MORPHING_OPT, ADMIN_ROLE_NAME, NEW_TENANT_NAME_OPT,
-    NEW_TENANT_NEUTRON_QUOTAS_OPT, NEW_TENANT_CINDER_QUOTAS_OPT,
-    NEW_TENANT_NOVA_QUOTAS_OPT, NEW_TENANT_ADMIN_USERS_OPT,
-    NEW_TENANT_OPEN_DEFAULT_SECGROUP_OPT, NEW_TENANT_SECGROUP_PROTOCOLS_OPT]
+    NEW_SECGROUP_NAME_OPT, NEW_TENANT_NEUTRON_QUOTAS_OPT,
```

Comment:
Why not  insert the new `NEW_SECGROUP_NAME_OPT` at the end of the list? (this way if someone does a `git blame` on these lines they'll see your seemingly unrelated secgroup commit for all 4 lines, instead of just one line, and they'll need to `git checkout` your commit~1 and `git blame` again to see the real commit(s) the rest of these options were added in)